librz/util/set.c

```diff
@@ -3,32 +3,32 @@
 
 #include <rz_util/set.h>
 
-// p
+// s
```

Comment:
Actually this can be removed? Or written out. No idea what the comment does there anyways.